WcaOnRails/app/assets/stylesheets/wca.scss

```diff
@@ -382,6 +382,8 @@ textarea {
 .event-radio {
   .cubing-icon {
     font-size: 2em;
+    margin: 0;
+    width: auto;
```

Comment:
Overrides the Semantic UI icon styles, so that the checkbox icons are displayed more densely (i.e. as they are currently).